### ISSUE TITLE
Strand-specific information and --UTR option.

### ISSUE THREAD

**Issue by aswathyseb** [Gaius-Augustus/BRAKER#311]
The manual says if --UTR=on, then strand-specific files can be provided. Does  that mean strand-specific files have no effect if --UTR=on option is not provided? 

Also, how should I specify strand information for TruSeq strand-specific libraries where read1 is the reverse complement of actual sequence? I separated the bam file into fwd.bam and rev.bam where fwd.bam corresponds reads mapping to genes in the + strand (where reads are mapping in the -ve strand or blue color reads in IGV) and vice versa for rev.bam. Is that correct?

In other words, does '+' corresponds to sense strand and '-' corresponds to antisense?

**Comment by KatharinaHoff** [comment]
That's what we meant, yes.
